ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+HOST_PROJECT="openshift-dev-installer"
+HOST_PROJECT_CONTROL_SUBNET="installer-shared-vpc-subnet-1"
+HOST_PROJECT_COMPUTE_SUBNET="installer-shared-vpc-subnet-2"
+HOST_PROJECT_NETWORK="installer-shared-vpc"
+BASE_DOMAIN="installer.gcpxpn.devcluster.openshift.com"
```

Comment:
`installer.gcpxpn.devcluster.openshift.com` is the base domain for the openshift shared vpc service project, which is what we use for our local/dev testing. That basedomain would be different in CI. I'm pretty sure you can actually just delete this line, and the appropriate BASE_DOMAIN will be populated in a different step...
But this is why the most recent rehearsal failed.